status/flowcells.py

```diff
@@ -76,12 +78,123 @@ def list_flowcells(self, all=False):
 
         return OrderedDict(sorted(temp_flowcells.items(), reverse=True))
 
+    def list_ont_flowcells(self):
+        ont_flowcells = OrderedDict()
+        view_all = self.application.nanopore_runs_db.view("info/all_stats", descending=True)
+        view_status = self.application.nanopore_runs_db.view("info/run_status", descending=True)
```

Comment:
Do we really need two views here? The data returned by view_all does not seem that much more than run_status. Wouldn't just view_all suffice?

Reply:
Would simplify a lot of the code below where it checks for ongoing or finished.

Reply:
Transfer "running"-vs-"finished"-logic to views instead of back-end.